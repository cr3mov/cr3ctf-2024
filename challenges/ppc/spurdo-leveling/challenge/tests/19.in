212888689998258452 942169645424396803
1 2108703
1 9164191
