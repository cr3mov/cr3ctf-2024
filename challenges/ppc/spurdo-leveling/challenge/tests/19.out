64693315731 64693315731 64693315731
